modules/dynamic-apiserver/pkg/config/config.go

```diff
@@ -32,8 +32,9 @@ type Config struct {
 	Cors           struct {
 		AllowedOrigins []string `envconfig:"default=https://*;http://*"`
 	}
-	SentryDsn string `envconfig:"optional"`
-	Domain    string
+	SentryDsn    string `envconfig:"optional"`
+	DeploymentId string `envconfig:""`
```

Comment:
Could DeploymentId set via config? Why?

Reply:
Here I only marked it optional, as agreed